modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,69 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding the `arm64` bootimage to your machine set 
+
+To deploy the heterogeneous cluster, you must add the generated `arm64` bootimage to your Azure machine set. To create your own custom machine set on Azure, see "Creating a machine set on Azure".
+
+.Prerequisites 
+
+* Installed the OpenShift CLI (`oc`).
+* Generated a custom Azure machine set. 
+
+.Procedure
+. Check that your machine set was created:
++
+[source,terminal]
+----
+oc get machineset
+----
+. Add your generated `arm64` bootimage to the `resourceID` parameter, and then edit `vmSize` to the instance type used in your installation: 
```

Comment:
Change the name in the generated yaml file, set the `resourceID` to the `arm64` bootimage, and then edit the `vmSize` to....